notebooks/StatiGraphics.jl

```diff
@@ -0,0 +1,164 @@
+### A Pluto.jl notebook ###
+# v0.12.20
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 01c88650-1fa3-11eb-1156-bb632230292b
+using Random; Random.seed!(2019); # make sure this tutorial is reproducible
+
+# ╔═╡ 221618e0-6e73-11eb-047e-3b329eefa7d3
+begin
+	using GeoStats
+	using StratiGraphics
+	using SpectralGaussianSimulation
+	
+	solver₁ = FFTGS(:land => (variogram=GaussianVariogram(range=100.,sill=3e-2),))
+	solver₂ = FFTGS(:land => (variogram=GaussianVariogram(range=100.,sill=3e-2),))
+	
+	procs = [GeoStatsProcess(solver) for solver in [solver₁, solver₂]];
+end
+
+# ╔═╡ cc7eef10-6e72-11eb-2cd4-abf33bd7f1ad
+md"""
+# Installation & Setup
+
+Before we proceed, please install the following packages:
+"""
+
+# ╔═╡ 6c9b41d8-043f-4626-8798-4a3ae9da922b
+md"""
+# Creating surfaces
+
+First, we demonstrate how Markov-Poisson sampling can be used to generate random surfaces of a 3D stratigraphic model.
+
+We start by defining a set of 2D geostatistical processes:
+"""
+
+# ╔═╡ 34e7730e-6e73-11eb-14e4-9f7bfe169470
+md"""
+In this example, we define two Gaussian processes. This processes represent smooth "depositional" processes as indicated by the GaussianVariogram model. The range and sill parameters determine the "frequency" and "height" of oscillation in the horizons, respectively.
+
+Having a set of processes defined to evolve a landscape, we need transition probabilities between these processes in order to build stratigraphy over time. Here we say that the two processes are balanced (i.e. 50% / 50% chance in the long run):
+"""
+
+# ╔═╡ 497c2820-6e73-11eb-3b09-4facec99f59a
+P = [0.5 0.5
+     0.5 0.5]
+
+# ╔═╡ 519a6760-6e73-11eb-2987-fd6e15456f15
+md"""
+Finally, we specify a duration process that determines the (random) amount of time that each process takes before a new transition:
+"""
+
+# ╔═╡ 6029f930-6e73-11eb-3834-a355dd9550fa
+ΔT = ExponentialDuration(1.0)
+
+# ╔═╡ 6778b370-6e73-11eb-03fa-738c64b466a1
+md"""
+We create a geological environment with the components defined above:
+"""
+
+# ╔═╡ 7bc5b120-6e73-11eb-29c1-7de2d8ab37c4
+env = Environment(procs, P, ΔT)
+
+# ╔═╡ 8c6fbe80-6e73-11eb-388f-bb64eaf84d96
+md"""
+Given an initial state (i.e. flat land), and a number of epochs, we can simulate the environment to produce a geological record:
+"""
+
+# ╔═╡ ab57bb40-6e73-11eb-208e-b76a2961ad6c
+begin
+	nepochs = 10 # number of surfaces to simulate
+	
+	init = LandState(zeros(500,500)) # each surface is a 500x500 image
+	
+	record = simulate(env, init, nepochs);
+end
+
+# ╔═╡ bb05dc70-6e73-11eb-0e35-25fb65c02fd4
+begin
+	using Plots; gr(format=:png)
```

Comment:
`using` statements should go in the header like in the other notebooks in this repository.

Reply:
Sure